control-plane-cf.template

```diff
@@ -420,15 +409,15 @@ Resources:
 
             - |
                 mkdir /opt/helm
-            - |2
+            - |
                 cd /opt/helm
-            - |2
+            - |
                 helm init -c
-            - |2
+            - |
                 helm repo add banzaicloud-stable http://kubernetes-charts.banzaicloud.com
-            - |2
+            - |
                 helm repo update
-            - |2
+            - |
                 helm repo list
             - |2
```

Comment:
Why did you remove the other |2 above and leave this one intact?

Reply:
We don't need two "space" indent in these lines. So it's ok, if we remove it.

Reply:
Thanks for the clarification:)